packages/terra-i18n/tests/jest/generate-aggregated-translations-directory.js

```diff
@@ -0,0 +1,11 @@
+const aggregateTranslations = require('../../scripts/aggregate-translations/aggregate-translations');
```

Comment:
We should use terra-toolkit's version of this as this script is deprecated

Reply:
We should run & mock this in before hooks instead of adding a pre-jest script

Reply:
Agreed, I [updated](https://github.com/cerner/terra-core/pull/1853/commits/9879ae9453e16626f24a8ef709862a70ba334596) this to use terra-toolkit's version of this script. I don't believe using before hook in test file will work as the before hook would come after [the import for the file](https://github.com/cerner/terra-core/blob/master/packages/terra-i18n/tests/jest/intlLoaders.test.js#L3) we'd need to generate in the before hook. The pre-jest script will go away, along with most all of the jest tests, once we remove the aggregate-translations script code from terra-i18n. I want to wait until we are ready to cut a major version bump on terra-i18n before we cut that though.